GVFS/GVFS/CommandLine/StatisticsVerb.cs

```diff
@@ -0,0 +1,298 @@
+﻿using CommandLine;
+using GVFS.Common;
+using GVFS.Common.Database;
+using GVFS.Common.FileSystem;
+using GVFS.Common.Git;
+using GVFS.Common.NamedPipes;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace GVFS.CommandLine
+{
+    [Verb(StatisticsVerb.StatisticsVerbName, HelpText = "Get statistics for the health state of the repository")]
+    public class StatisticsVerb : GVFSVerb.ForExistingEnlistment
+    {
+        private const string StatisticsVerbName = "statistics";
+
+        private const int DirectoryDisplayCount = 5;
+
+        [Option(
+            'c',
+            "crawl",
+            Required = false,
+            HelpText = "Hydrate all files (for testing purposes)")]
+        public bool ToCrawl { get; set; }

```

Comment:
For the product version of this, let's not include test code (or scaffolding code) that would not be used by end users.

Reply:
I plan to write another commit once the review is done that removes debugging tools/code and the commented code to actually be put in the product version. I'm including everything here so that reviewers have as many tools to debug as needed and I thought it might be nice, as well as the background information about why I'm doing it the way that I am currently.